recipes/tassel/build.sh

```diff
@@ -0,0 +1,14 @@
+BINARY_HOME=$PREFIX/bin 
+TASSEL_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+# Copy source to the conda environment
+mkdir -p $TASSEL_HOME 
+cp -R $SRC_DIR/* $TASSEL_HOME/
+
+# Convert CRLF in wrappers to LF
+sed -i $'s/.$//' $TASSEL_HOME/*.pl
```

Comment:
Does the leading `$` for the `sed` have any purpose?
However, maybe use
```bash
sed -i 's/\r$//' $TASSEL_HOME/*.pl
```
instead. That way the `sed` won't cause any trouble in case the TASSEL developers at some point in the future decide to use Unix line endings.